7-layouts/style.css

```diff
@@ -0,0 +1,135 @@
+
+@media screen and (min-width: 545px){
+    h1 {
+       font-size: 400%;
+   }
+
+    h2 {
+        font-size: x-large;
+        width: 40%;        
+        padding-bottom: 3%;
+    }
+
+    p {
+        font-size: 130%;
+    }
+}
+
+@media screen and (max-width: 1024px){    
+    article {
+        width: 40%;   
+    }  
+    
+   h1 {
+        font-size: xx-large;
+    }
+}
+
+@media screen and (max-width: 544px){
+    article {
+       width: 70%;  
+   }   
+
+    h1 {
+       
+       font-size: 180%;
+    }
+
+    h2 {
+        font-size: x-large;
+        width: 70%;
+        padding-bottom: 6%;
+    }    
+
+    p {
+        font-size: 70%;
+    }   
+}
+
+@media screen and (min-width: 869px){
+    article {
+        padding-left: 10px;
+        padding-right: 10px;
+        width: 30%;    
+    }
+
+    h2 {
+        font-size: x-large;
+        width: 30%;
+    }
+}
+
+article {
+    padding-left: 15px;
+    padding-right: 15px;
+}
+
+
+.banner {
+    align-items: center;
```

Comment:
is this property being used?

Reply:
Not really, but I changue te property to justify-content and I can see the change